frontend/helpers/product-list-helpers.ts

```diff
@@ -58,12 +55,11 @@ export function encodeDeviceTitle(deviceTitle: string): string {
 type ProductListPathAttributes = Pick<
    ProductList,
    'type' | 'handle' | 'deviceTitle'
-> & {
-   itemType?: string;
-};
+>;
 
 export function getProductListPath(
-   productList: ProductListPathAttributes
+   productList: ProductListPathAttributes,
+   itemType?: string
```

Comment:
If we don't actually pass itemType at any callsites, maybe we can drop this param

Reply:
https://github.com/iFixit/react-commerce/pull/431/commits/198aa5a9e132b038ff6a0610908984d2361fac69